src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderDropdown.scss

```diff
@@ -16,6 +16,29 @@
   z-index: 0;
 }
 
+.global-header--list {
+  /* Works on Firefox */
+  scrollbar-width: thin !important;
```

Comment:
The use of `!important!` actually comes up in two stylesheets in the PR, so here's how I resolved it:

1. This particular class didn't actually need the `!important`. It can be safely removed.

2. Elsewhere, it was necessary to force the width property on the portion of the `GlobalHeader` component relying on `Dropdown` from clockface. After playing around it with, the width property _is_ getting correctly populated to the `Dropdown` via the classname, but that actually causes a separate issue, which is that both the parent component and the `Dropdown` have the same style, so the margin and padding would have to be separately adjusted for each with different class names in order for us to get the right width.

Simpler solution was to add this into a style outside the component, since there are only two properties being adjusted, so that's what I ended up proposing in my new commit.